.github/workflows/win-build.yml

```diff
@@ -58,8 +58,8 @@ jobs:
         uses: trufflesecurity/trufflehog@main
         with:
           path: ./
-          base: ${{ github.event.repository.default_branch }}
-          head: HEAD
+          base: ""
+          head: ${{ github.ref_name }}
```

Comment:
This comes from [TruffleHog's documentation for advanced usage to scan an entire branch](https://github.com/trufflesecurity/trufflehog?tab=readme-ov-file#advanced-usage-scan-entire-branch).